### PR TITLE
New Component last artist, created and finished

### PR DESCRIPTION
**Pull request by Elfaram**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -2,17 +2,18 @@ import Navbar from "@components/Navbar";
...
-      <Route path="/" element={<Home />} />
-      <Route path="/profile/:profileId" element={<Profile />} />
-       </Routes>
-     </div>
-
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/profile/:profileId" element={<Profile />} />
+        <Route path="/RegisterArtistLast" element={<Register />} />
```
Comment (LoicBrassart) [inline frontend/src/App.jsx]: I don't recommend uppercase characters in urls

File: frontend/src/components/RegisterArtistLast/index.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+export default function RegisterArtistLast() {
+  const [siren, setSiren] = useState("");
+  const [selection, setSelection] = useState("");
+
+  const hChangeSiren = (evt) => {
+    setSiren(evt.target.value);
+  };
+  const hChangeSelection = (evt) => {
+    setSelection(evt.target.value);
+  };
```
Comment (LoicBrassart) [inline frontend/src/components/RegisterArtistLast/index.jsx]: Please refactor these functoins

File: frontend/src/components/RegisterArtistLast/index.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        <div className="registerContainer">
+          <h1>Employeur</h1>
+          <form className="registerForm">
+            <select className="inputForm" onChange={hChangeSelection}>
+              <option value={selection} disabled selected hidden>
+                Statut...
+              </option>
+              <option value="0">Intermittent du spectacle </option>
+              <option value="1">Artiste-auteur </option>
+              <option value="2">Auto-entrepreneur </option>
```
Comment (LoicBrassart) [inline frontend/src/components/RegisterArtistLast/index.jsx]: Use more meaningful values for your forms

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some evils to fix

**Review by Elfaram — COMMENTED** [review]
changes FormData and Values

**Review by ghost — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

**Comment by LoicBrassart** [comment]
Solve conflicts plz

**Review by ghost — DISMISSED** [review]

**Comment by ghost** [comment]
solve conflict

**Review by ghost — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/components/RegisterArtistLast/index.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+import SRegisterArtistLast from "./style";
+
+export default function RegisterArtistLast() {
+  const [formData, setFormData] = useState({
+   siren: "",
+    status: "",
```
Comment (LoicBrassart) [inline frontend/src/components/RegisterArtistLast/index.jsx:7]: Indentation

**Review by LoicBrassart — APPROVED** [review]
Stop editing on GHub so often!

**Pull request merged**